pkgs/top-level/all-packages.nix

```
@@ -18130,6 +18130,8 @@ in
 
   browsh = callPackage ../applications/networking/browsers/browsh { };
 
+  brotab = callPackage ../tools/misc/brotab { };
```

related to other change :)
```suggestion
  brotab = callPackage ../tools/misc/brotab {
    python = python3;
  };
```